eris.go

```diff
@@ -164,6 +169,19 @@ func (e *wrapError) Format(s fmt.State, verb rune) {
 	printError(e, s, verb)
 }
 
+// StackFrames returns a slice of all the frames in the error chain
+func (e *wrapError) StackFrames() []uintptr {
+	frames := *e.stack
+	switch nextErrType := e.Unwrap().(type) {
+	case *wrapError:
+		frames = append(frames, nextErrType.StackFrames()...)
```

Comment:
this method needs to change slightly so that wrap error layers are inserted into the root error stack (similar to how Unpack currently works)